aldryn_faq/tests/frontend/integration/pages/page.faq.crud.js

```diff
@@ -11,7 +12,45 @@
 var faqPage = {
     site: 'http://127.0.0.1:8000/en/',
     mainElementsWaitTime: 12000,
-    iframeWaitTime: 15000
+    iframeWaitTime: 15000,
+
+    // log in
+    editModeLink: element(by.css('.inner a[href="/?edit"]')),
+    usernameInput: element(by.id('id_cms-username')),
+    passwordInput: element(by.id('id_cms-password')),
+    loginButton: element(by.css('.cms_form-login input[type="submit"]')),
+    userMenus: element.all(by.css('.cms_toolbar-item-navigation > li > a')),
+    testLink: element(by.css('.selected a')),
+
+    cmsLogin: function (credentials) {
+        // object can contain username and password, if not set it will
+        // fallback to 'admin'
+        credentials = credentials ||
+            { username: 'admin', password: 'admin' };
+
+        faqPage.usernameInput.clear();
+
+        // fill in email field
+        faqPage.usernameInput.sendKeys(
+            credentials.username).then(function () {
+            faqPage.passwordInput.clear();
```

Comment:
It goes after ```faqPage.usernameInput.sendKeys(credentials.username).then(function () {```, so should be moved on one tabspace regarding ```faqPage.usernameInput.sendKeys(``` (as it is now), right?